ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/item/complex/SurveyDetail.java

```diff
@@ -56,6 +57,21 @@ public void setCameraDetail(CameraDetail cameraDetail) {
         this.cameraDetail = cameraDetail;
     }
 
+    /**
+     * True if aircraft's yaw is locked to the angle of the survey (added: client-lib:v3.0)
```

Comment:
@TShapinsky to describe when the method was added, use the `@since` javadoc tag, as done [here](https://github.com/dronekit/dronekit-android/blob/669e3f98ec511aeb1649264deb517bc311233031/ClientLib/src/main/java/com/o3dr/android/client/apis/VehicleApi.java#L259).